### PR TITLE
Add partial solutions for Chapter 1

### PR DESCRIPTION
**Pull request by canyon289**
Let me know if this is in the right direction, formatting and everything. I can make whatever changes you like

### CODE REVIEW

**Comment by aloctavodia** [comment]
Thanks @canyon289 this is great! 

I have a few comments

* I may burn in space-hell but are you sure the are two Popes in Futurama?
* You have a missing prior in Q3
* You are right about Q5
* The reasoning in Q6 is right, but you have a typo-like mistake. You wrote $p(heads \mid biased) = 1$ twice

File: exercises/Chapter1.ipynb [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The priors in thie model are\n",
+    "\n",
+    "$$ \\mu \\text{~} Normal(0,10)$$\n",
+    "$$ \\sigma \\text{~} HalfNormal(25)$$\n",
```
Comment (canyon289) [inline exercises/Chapter1.ipynb]: @aloctavodia The two priors are listed here, I can see them in my local Jupyter notebook, I'm not sure why its not rendering it Giithub. Will investigate

**Comment by canyon289** [comment]
> Thanks @canyon289 this is great!
> 
> I have a few comments
> 
> * I may burn in space-hell but are you sure the are two Popes in Futurama?
> * You have a missing prior in Q3
> * You are right about Q5
> * The reasoning in Q6 is right, but you have a typo-like mistake. You wrote $p(heads \mid biased) = 1$ twice

* For the pope in Futurama I actually don't know haha. I'll do some googling
* For Q3 I included both priors but I don't know why its not showing on github. Will look into it
* Will fix

I'll continue on with the rest of the questions in Chapter 1 and do one PR per chapter

**Comment by canyon289** [comment]
@aloctavodia Ready for review.

Two questions
1. Did I understand the Space Pope question correctly this time?
2. I don't have answers for 8 onwards. These questions were more user experimentation than a specific "correct" answer. Should something be included or can these be omitted entirely?

**Comment by aloctavodia** [comment]
LGTM!

The only comment I have, is just a detail. I would not say that "the is no overlap between popes and humans in Futurama". There is no overlap between the reptilian Pope and humans, but after the reptilian pope resigns or die a human may have the chance to become pope, at least this is my understating of liturgy of the First Amalgamated Church, haha.

**Comment by aloctavodia** [comment]
BTW, you are right some of the questions are about user experimentation. So probably is OK to omit those. Maybe if some else contribute answer in the future they may want to add something to those answers.

**Comment by aloctavodia** [comment]
@canyon289 thank you very much!

**Comment by canyon289** [comment]
Definitely! I'm going to work through the remaining chapters in the coming days/weeks

**Pull request merged**